### ISSUE TITLE
Adding multiple labels with ionosctl k8s nodepool update

### ISSUE THREAD

**Issue by vquicksilver** [ionos-cloud/ionosctl#164]
## Description
The documentation is not clear about how we can add multiple labels to a nodepool using ionosctl, the API seems to support it though, can we use ionosctl for adding multiple labels? if so how? Running something like:

$ ionosctl k8s nodepool update --cluster-id id --nodepool-id id --label-key label1 --label-value value1 --label-key label2 --label-value value2

doesn't work, it justs add the last label2.

Running it multiple times trying to add a label at a time doesn't work either, the last label is replaced by the new one.

## Expected behavior
Multiple labels are added to the nodes in the nodepool instead of just one.

## Environment

Go Version: 
```
```
Ionosctl version: 
ionosctl version 6.1.3
```
OS: 
```
```
Shell: 
```
```

## Configuration Files
<!-- could also be environment settings, e.g. LANG=de_DE -->

## How to Reproduce
<!--
Steps to reproduce the behavior:
1. Go to '...'
2. Execute `acme --emca template.xml`
3. ...
Might be omitted if there are no choices, eg. the single command is already in the description
-->

## Error and Debug Output
<!--
Add anything, that could help here.
You can increase the debug level by .. ' hello --whatsover=example`
-->

## Additional Notes
<!--
Add any other content about the problem here.
-->

## References
<!--
Add any other related issues or other links that you think are important for fixing the bug.
-->

**Comment by AnaMariaMoga** [comment]
Hello,

Thank you for opening this. For the moment, it seems like adding multiple labels (or annotations ) using `ionosctl k8s nodepool` is not supported. We will have a look and come back with updates.

Ana

**Comment by AnaMariaMoga** [comment]
Hi,

This issue has been fixed in the [v6.1.5](https://github.com/ionos-cloud/ionosctl/releases/tag/v6.1.5) release. You can add multiple labels to the k8s nodepool using:
```
ionosctl k8s nodepool update --cluster-id id --nodepool-id id --labels key1=value1,key2=value2
```
Also, if you want to set the labels from the beginning, you can use the same flag and format for `ionosctl k8s nodepool create` command.

**Comment by AnaMariaMoga** [comment]
I will close this task; if there are any other issues regarding this, feel free to reopen it or open a new one.
